entity-types/ext-istio_ingress/tests/Metric.json

```diff
@@ -0,0 +1,54 @@
+[
```

Comment:
there is no `cluster_name` in this datapoint which is making the test fail.

All the attributes in the identifier are required in the datapoint